oak_functions/loader/src/server.rs

```diff
@@ -1024,6 +1030,20 @@ mod tests {
         assert_eq!(ChannelStatus::ChannelHandleInvalid, result.unwrap_err())
     }
 
+    #[tokio::test]
+    async fn test_drop_channel_switchboard_stops_recv() {
+        let mut channel_switchboard = ChannelSwitchboard::new();
+        let mut endpoint_2 = channel_switchboard.register(ChannelHandle::Testing);
+
+        let stopped_receiving = tokio::spawn(async move {
+            endpoint_2.receiver.recv().await;
+            true
+        });
+
+        std::mem::drop(channel_switchboard);
```

Comment:
Could this be in an inner scope and just be dropped naturally? Just so it's closer to what actually happens, but feel free to ignore if it complicatees things.